python/bridgestan/model.py

```diff
@@ -685,6 +674,58 @@ def _handle_error(self, err: ctypes.c_char_p, method: str) -> Exception:
 
         return RuntimeError(f"Unknown error in {method}. ")
 
+    @classmethod
```

Comment:
I'm still on the page of putting a deprecation warning in the function doc.  What do you think of this idea in general?  I'll stop pointing it out for the rest of this review.